misc/afl-libfuzzer/afl-libfuzzer-daemon.py

```diff
@@ -555,6 +561,14 @@ def scan_crashes(base_dir, collector, cmdline_path=None, env_path=None, test_pat
             if base_env:
                 env = dict(base_env)
 
+            submission = crash_file
+            if transform:
+                try:
+                    submission = apply_transform(transform, crash_file)
+                except Exception as e:
```

Comment:
I think we shouldn't die on exceptions here without submitting at least something that might make it possible to debug why the transform failed. We should probably send at least the untransformed testcase before dying.

Reply:
I removed the return statement from these failed attempts.  Instead of exiting, we warn and the original testcase will still be submitted.